local-cli/bundle.js

```diff
@@ -0,0 +1,65 @@
+var http = require('http');
+var fs = require('fs');
+var path = require('path');
+var chalk = require('chalk');
+var blacklist = require('../packager/blacklist.js');
+var ReactPackager = require('../packager/react-packager');
+
+var OUT_PATH = 'iOS/main.jsbundle';
+
+function getBundle(flags) {
+
+  var options = {
+    projectRoots: [path.resolve(__dirname, '../../..')],
+    transformModulePath: require.resolve('../packager/transformer.js'),
+    assetRoots: [path.resolve(__dirname, '../../..')],
+    cacheVersion: '2',
+    blacklistRE: blacklist('ios')
+  };
+
+  var url = '/index.ios.bundle?dev=' + flags.dev;
+
+  console.log('Building package...');
+  ReactPackager.buildPackageFromUrl(options, url)
+    .then(function(bundle) {
```

Comment:
nit: use `done` to make sure if `ReactPackager.buildPackageFromUrl` throws the error is reported.